index.html

```diff
@@ -126,15 +126,65 @@ <h3 class="card_title ">Project name goes here</h3>
                 </div>
             </div>
     </section>
-
-
-    </div>
     <div class="about">
         <h2>About</h2>
         <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
         <a href="" class="button"><span>See this project</span>
         <img src="images/icon1.svg" height="16px" ></a>
     </div>
+    <section class="skills_section">
+        <div class="language">
+            <h3>Languages</h3>
+            <ul>
+                <li>JavaScript(ES5/ES6)</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+            </ul>
+        </div>
+        <div class="frameworks">
+            <h3>Frameworks</h3>
+            <ul>
+                <li>Bootstrap</li>
+                <li>Ruby on rails</li>
+                <li>RSpec</li>
+                <li>Capibara</li>
+                <li>Selenium</li>
+            </ul>
+        </div>
+        <div class="skills">
+            <h3>Skills</h3>
+            <ul>
+                <li>Codekit</li>
+                <li>Codepen</li>
+                <li>GitHub</li>
+                <li>Gitlab</li>
+                <li>Terminal</li>
+            </ul>
+        </div>
+    </section>
+    <section class="contact_section">
+        <div class="contact-page">
+            <img src="images/image1.png" height="100px">
+            <div>
+                <h2>Get started</h2>
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ea eiusmod</p>
+            </div>
+            <div class="form_section">
+                <form action="https://formspree.io/f/xyyvlwnp" method="post">
+
+                    <input type="text" class="user-name" name="user_name" placeholder="Full-name" maxlength="30" required/>
+
+                    <input type="email" class="mail" name="user_email" placeholder="Email Address" required placeholder="username@example.com" pattern=".+@exampl\.com" title="Please enter correct e-mail address" />
+
```

Comment:
- [ ] Please make your that your email address accepts valid emails. As you can see from the screenshot below the email field does not accept my valid email. Due to that, I couldn't submit the form :smile: I think it is because of the `pattern` attribute you gave to your email field so please kindly fix it. :pray: 
![Screenshot from 2022-12-06 23-33-59](https://user-images.githubusercontent.com/92303891/206048649-0e921b1a-f980-443e-8d4d-84e35a66d1bb.png)